### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by ggboypro** [community/community#113933] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi, Contact support. You are currently logged in as xxxxxx.

```
2024-03-24 09:25:22.875 [info] [auth] Invalid copilot token: missing token: 403 
```

**Comment by mmqeth** [comment]
Check this:
https://github.com/orgs/community/discussions/113106#discussion-6389052

It seems that github had a recent change to the copilot plan that was not handled properly

**Comment by davecheney** [comment, marked as answer]
@powerttt 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
